### ISSUE TITLE
Post original exception when module was not found in bin/addons-linter

### ISSUE THREAD

**Issue by EnTeQuAk** [mozilla/addons-linter#1617]
I merged a branch with master and didn't update dependencies so when I ran `bin/addons-linter` it did throw the warning that I apparently did not yet build addons-linter.

So we should at least post the original error message or make the check in `bin/addons-linter` more intelligent.

**Comment by Divya063** [comment]
@EnTeQuAk Can I work on this?

**Comment by EnTeQuAk** [comment]
Absolutely! 😃  Just let me know once you start working on it so that I can set the correct status / label.

**Comment by Divya063** [comment]
@EnTeQuAk I'm working on it, I just want to know whether a new message should be posted or the checks should be modified?

**Comment by EnTeQuAk** [comment]
@Divya063 I like the new message. More details are always better. I'd still do a `console.log` of the actual error message separated by a few empty lines after that error message to make sure the user has the ability to act accordingly.

**Comment by Divya063** [comment]
@EnTeQuAk I'm thinking to add the error message in this way
```javascript
 console.log('You did not build addons-linter yet.');
 console.log('Please run yarn and yarn build to update the dependencies');
 console.log('If you are a new user, see README.md for more information');


```
